db/db_filesnapshot.cc

```diff
@@ -372,8 +372,17 @@ Status DBImpl::GetLiveFilesStorageInfo(
 
   size_t wal_size = live_wal_files.size();
```

Comment:
I think this includes both: wal files required by manifest file, and all the other lingering ".log" files that still exist by the time the snapshot is taken. If we wan to differentiate them, maybe log in `GetSortedWalFiles` to break it down further?

Reply:
Sure - fixed